### PR TITLE
[fix bug 1449758] Landing page for Facebook Container

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds a landing page for the Facebook Container extension. Non-Firefox gets a download CTA, Firefox gets a link to the extension on AMO.

No l10n for now but we may decide to localize the page later.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1449758

## Testing
https://bedrock-demo-craigcook.us-west.moz.works/firefox/facebookcontainer/

### CODE REVIEW

**Comment by craigcook** [comment]
This is ready for an initial code review but marking DNM for the moment pending final signoff by all parties.

**Comment by craigcook** [comment]
Got signoff in the bug, this is good to go.

File: bedrock/firefox/templates/firefox/facebookcontainer/index.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/facebookcontainer" %}
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/facebookcontainer/index.html]: As we aren't localizing yet, will this confuse L10n? (Ignore if this has already been discussed with the L10n team.)

File: media/js/firefox/facebook-container-video.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
```
Comment (jpetto) [inline media/js/firefox/facebook-container-video.js]: Should have `$` in the parens here as the internal/sandboxed name for `window.jQuery` (passed in below).

File: media/js/firefox/facebook-container-video.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    'use strict';
+
+    var $html = $(document.documentElement);
+    var videoPoster = new Mozilla.VideoPosterHelper('.fbcontainer-video');
+
+    if (window.Mozilla.Client.isFirefox) {
+        $html.addClass('is-firefox');
+    } else {
+        $html.addClass('is-not-firefox');
+    }
```
Comment (jpetto) [inline media/js/firefox/facebook-container-video.js]: Doesn't look like this `else` condition is used - can we remove it?

File: bedrock/firefox/templates/firefox/facebookcontainer/index.html (hunk 1 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/facebookcontainer/index.html]: I'm not sure of the longer-term l10n plan or how long this page will live. I'll remove the lang file and if we decide to localize it later we can always add it back.

File: media/js/firefox/facebook-container-video.js (hunk 2 above)
Comment (craigcook) [reply media/js/firefox/facebook-container-video.js]: Added!

File: media/js/firefox/facebook-container-video.js (hunk 3 above)
Comment (craigcook) [reply media/js/firefox/facebook-container-video.js]: Removed!

File: bedrock/firefox/templates/firefox/facebookcontainer/index.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+  {% block details %}
+  <section class="details content-section">
+    <div class="content">
+      <ul>
+        <li>
+          <h3>{{ _('Opt out on your terms') }}</h3>
+          <p>
+          {% trans fbcontainer='https://addons.mozilla.org/firefox/addon/facebook-container/' %}
+            Facebook can track almost all your web activity and tie it to your facebook identity. If that’s too much for you, the <a href="{{ fbcontainer }}">Facebook Container extension</a> isolates your identity into a separate container tab, making it harder for Facebook to track you on the web outside of Facebook.
```
Comment (peiying2) [inline bedrock/firefox/templates/firefox/facebookcontainer/index.html]: facebook - > Facebook.

**Pull request merged**